Update NDR for nutrient load type feature (#181)
natcap/invest.users-guide: source/en/ndr.rst

    .. note::
       Loads are the sources of nutrients associated with each LULC class. This value is the total load from all sources. If you want to represent different levels of fertilizer application, you will need to create separate LULC classes, for example one class called "crops - high fertilizer use" a separate class called "crops - low fertilizer use" etc.

    - :investspec:`ndr.ndr biophysical_table_path.columns.nut_load_type`

    .. note::

>        Data sources may provide loading values as either the amount of applied nutrient (e.g. fertilizer, livestock waste, atmospheric deposition); or as “extensive” measures of contaminants, which are empirical values representing the contribution of a parcel to the nutrient budget (e.g. nutrient export running off urban areas, crops, etc.). The two types are denoted *application-rate* and *measured-runoff* respectively in the biophysical table. In the case of having applied nutrient values, the model will correct for the nutrient retention provided by the pixel itself, using the application rate and retention efficiency value (*eff_n* or *eff_p*) for that land cover type:

Reviewer: ```suggestion
       Data sources may provide loading values as either the nutrient application rate (e.g. fertilizer, livestock waste, atmospheric deposition); or as measured contaminant runoff, which are empirical values representing the contribution of a parcel to the nutrient budget (e.g. nutrient export running off urban areas, crops, etc.). These two types of loading values are denoted *application-rate* and *measured-runoff* respectively in the biophysical table. In the case of having application rate values, the model will adjust for the nutrient retention provided on the pixel itself, using the application rate and retention efficiency value (*eff_n* or *eff_p*) for that land cover type:

```

>        applied_nutrient * (1 - retention_efficiency)

Reviewer: I'd like to make some clarifying edits to this paragraph and equation so the two loading options, and the words and fields that are used to describe them, are clear and consistent. One example of this would be changing the equation to something like

> _loading = application rate * (1 - retention efficiency)_
> Using the specific biophysical table fields:
> _loading = application-rate * (1 - [eff_n or eff_p])_

>        applied_nutrient * (1 - retention_efficiency)

Reviewer: ```suggestion
       _loading = application rate * (1 - retention efficiency)_
Using the specific biophysical table fields:
       _loading = application-rate * (1 - [eff_n_ or _eff_p])_

```

>        For example, if the nitrogen application rate for an agricultural LULC class is 10 kg/ha/year, and the retention efficiency is 0.4, the model will adjust the value to 6.0. If you have "extensive"/nutrient export values, denoted as *measured-runoff* in the biophysical table, then the model will use these directly without correction.

Reviewer: I'd suggest spelling out the adjustment, so

> ... the model will adjust the value to 6.0 kg/ha/year (= 10 kg/ha/year - (1 - 0.4))

>        For example, if the nitrogen application rate for an agricultural LULC class is 10 kg/ha/year, and the retention efficiency is 0.4, the model will adjust the value to 6.0. If you have "extensive"/nutrient export values, denoted as *measured-runoff* in the biophysical table, then the model will use these directly without correction.

Reviewer: ```suggestion
       For example, if the nitrogen application rate for an agricultural LULC class is 10 kg/ha/year, and the retention efficiency is 0.4, the model will adjust the value to 6.0 kg/ha/year (= 10 kg/ha/year * (1 - 0.4)). If you have measured/nutrient export values, denoted as *measured-runoff* in the biophysical table, then the model will use these directly without adjustment.

```

-------------
For all water quality parameters (nutrient load, retention efficiency, and retention length), local literature should be consulted to derive site-specific values. The NatCap nutrient parameter database provides a non-exhaustive list of local references for nutrient loads and retention efficiencies: https://naturalcapitalproject.stanford.edu/sites/g/files/sbiybj9321/f/nutrient_db_0212.xlsx. Parn et al. (2012) and Harmel et al. (2007) provide a good review for agricultural land in temperate climate.

> Data sources may provide loading values as either the amount of applied nutrient (e.g. fertilizer, livestock waste, atmospheric deposition); or as “extensive” measures of contaminants, which are empirical values representing the contribution of a parcel to the nutrient budget (e.g. nutrient export running off urban areas, crops, etc.) In the case of having applied nutrient values, the model will correct for the nutrient retention provided by the pixel itself, using the application rate and retention efficiency value (*eff_n* or *eff_p*) for that land cover type.

Reviewer: ```suggestion
Data sources may provide loading values as either the amount of applied nutrient (e.g. fertilizer, livestock waste, atmospheric deposition); or as “extensive” measures of contaminants, which are empirical values representing the contribution of a parcel to the nutrient budget (e.g. nutrient export running off urban areas, crops, etc.) In the case of having applied nutrient values, the model will correct for the nutrient retention provided by the pixel itself, using the application rate and retention efficiency value (*eff_n* or *eff_p*) for that land cover type.

Data sources may provide loading values as either the nutrient application rate (e.g. fertilizer, livestock waste, atmospheric deposition); or as measured contaminant runoff, which are empirical values representing the contribution of a parcel to the nutrient budget (e.g. nutrient export running off urban areas, crops, etc.). In the case of having application rate values, the model will correct for the nutrient retention provided on the pixel itself, using the application rate and retention efficiency value (*eff_n* or *eff_p*) for that land cover type.
```

> application_load * (1 - retention_efficiency)

Reviewer: Same comment as with the above paragraph that is similar/the same:

I'd like to make some clarifying edits to this paragraph and equation so the two loading options, and the words and fields that are used to describe them, are clear and consistent. One example of this would be changing the equation to something like

>     loading = application rate * (1 - retention efficiency)
>     Using the specific biophysical table fields:
>     loading = application-rate * (1 - [eff_n or eff_p])

> For example, if the nitrogen application rate for an agricultural LULC class is 10 kg/ha/year, and the retention efficiency is 0.4, the model will adjust the value to 6.0. If you have "extensive"/export values, then you may use them directly in the biophysical table without correction.

Reviewer: Same comment as with the previous paragraph that has the same text:

I'd suggest spelling out the adjustment, so

>    ... the model will adjust the value to 6.0 kg/ha/year (= 10 kg/ha/year - (1 - 0.4))

> For example, if the nitrogen application rate for an agricultural LULC class is 10 kg/ha/year, and the retention efficiency is 0.4, the model will adjust the value to 6.0. If you have "extensive"/export values, then you may use them directly in the biophysical table without correction.

Reviewer: ```suggestion
For example, if the nitrogen application rate for an agricultural LULC class is 10 kg/ha/year, and the retention efficiency is 0.4, the model will adjust the value to 6.0 kg/ha/year (= 10 kg/ha/year * (1 - 0.4)). If you have measured/nutrient export values, denoted as *measured-runoff* in the biophysical table, then the model will use these directly without adjustment.

```
